docs/source/install/min-prod-hw.rst

```diff
@@ -30,3 +30,40 @@ Prometheus uses more memory when querying over a longer duration (e.g. looking a
 
 For Prometheus alone, you should have 60MB of memory per core in the cluster and it would use about 600MB of virtual memory per core.
 Because Prometheus is so memory demanding, it is a good idea to add swap, so queries with a longer duration would not crash the server.
+
+.. raw:: html
+
+    <script>
+        function myFunction() {
+            const hosts = parseInt(document.getElementById('hosts').value);
+            const shards = parseInt(document.getElementById('shards').value);
+            const retention = parseInt(document.getElementById('retention').value);
+            let memory= (hosts*shards*60);
+            let disk = (hosts*shards*retention*12);
+
+            if (!Number.isNaN(disk)) {
+                disk = (disk > 1024)?((disk/1024).toFixed(2) + "GB") : disk.toString() + "MB";
+                document.getElementById('disk').textContent = disk;
+            }
+            if (!Number.isNaN(memory)) {
+                memory = (memory > 1024)?((memory/1024).toFixed(2) + "GB") : memory.toString() + "MB";
+                document.getElementById('memory').textContent = memory;
+            }
+            
+        }        
+    </script>
+    <div>
+    <table>
+    <colgroup>
+    <col style="width: 20%">
+    <col style="width: 20%">
+    <col style="width: 20%">
+    <col style="width: 20%;text-align: center;">
+    <col style="width: 20%;text-align: center;">
+    </colgroup>
+    <tr><th># Hosts</th><th>Thread Per Host</th><th>Retention</th><th>Memory</th><th>disk</th></tr>
```

Comment:
I sugget to use "Scylla Nodes" not hosts
"Cores pre node" not threads

For the the result, add a title "Prometheus" on top of Memory and Disk 
(better use Storage)

Reply:
Better to have a default for Scylla Nodes (3) and for Cores per Scylla Nodes (16)